src/main/java/lotto/ui/ResultView.java

```diff
@@ -11,13 +13,14 @@ public static void printTicket(String ticketLottoNumbers) {
         System.out.println(ticketLottoNumbers);
     }
 
-    public static void printGameResult(Map<Integer, Integer> score, double earningRate) {
+    public static void printGameResult(Map<Rank, Integer> score, double earningRate) {
         System.out.println("당첨 통계");
         System.out.println("---------");
-        System.out.println(String.format("3개 일치 (5000원)- %d개", score.get(3) == null ? 0 : score.get(3)));
-        System.out.println(String.format("4개 일치 (50000원)- %d개", score.get(4) == null ? 0 : score.get(4)));
-        System.out.println(String.format("5개 일치 (1500000원)- %d개", score.get(5) == null ? 0 : score.get(5)));
-        System.out.println(String.format("6개 일치 (2000000000원)- %d개", score.get(6) == null ? 0 : score.get(6)));
+        System.out.println(String.format("3개 일치 (%d원)- %d개", Rank.FIFTH.getWinningMoney(), score.get(Rank.FIFTH) == null ? 0 : score.get(Rank.FIFTH)));
```

Comment:
```suggestion
        System.out.println(String.format("3개 일치 (%d원)- %d개", Rank.FIFTH.getWinningMoney(), score.getOrDefault(Rank.FIFTH, 0)));
```
getOrDefault를 사용하면 더 간략하게 표현할 수 있어요.

또 이 아래를 보면 3개 일치, 4개 일치, ... 6개 일치 순으로 이어지는데요,
이 부분은 반복문으로 표현해도 괜찮지 않을까요?
`%d개 일치 (%d원)- %d개` 로도 표현할 수 있습니다.

Reply:
getOrDefault 를 적용하고, 반복문으로 바꾸었더니 의미가 조금 더 잘 전달되는 코드가 된 것 같습니다 👍